app/Main.hs

```diff
@@ -96,16 +96,18 @@ printDirectory (DirectoryPrefix dir) = do
     setSGR   [Reset]
 
 printTermResults :: Int -> (String, TermResults) -> IO ()
-printTermResults w (_, results) = do
+printTermResults w (_, results) =
     printMatches w results $ matches results
 
+likelihoodColor :: RemovalLikelihood -> Color
+likelihoodColor High = Red
+likelihoodColor Medium = Yellow
+likelihoodColor Low = Green
+
 printMatches :: Int -> TermResults -> [TermMatch] -> IO ()
-printMatches w _r ms = do
-    mapM_ printMatch ms
-  where
-    termFormat = "%-" ++ (show w) ++ "s"
-    printMatch m = do
-        setSGR [SetColor Foreground Dull Red]
+printMatches w r ms = do
```

Comment:
you shouldn't need `do` here since everything is under the `do` on the next line.